src/cli/README_SRP_SERVER.md

```diff
@@ -149,7 +149,7 @@ Done
 
 Usage: `srp server state`
 
-Print the state of the SRP server.
+Print the state of the SRP server. It could be `disabled`, `stopped` or `running`.
```

Comment:
May explain per state briefly? 
Specially, `running` -  active SRP server that would handle service registration. `stopped` - enabled but not active SRP server, eligible to become active SRP server spontaneously when detecting the lost of some active SRP server in Thread network.

Should be useful for some users (/testers) who may be not aware of implementation details and only refers to this README